doc/cmp.txt

```diff
@@ -366,6 +366,16 @@ enabled~
   `boolean | fun(): boolean`
   Toggles the plugin on and off.
 
+                                                *cmp-config.performance.debounce*
+performance.debounce~
+  `number`
+  Sets debounce time
+
+                                                *cmp-config.performance.throttle*
+performance.throttle~
+  `number`
+  Sets throttle time
```

Comment:
ditto

This value is used for throttling `filtering` and `displaying` results.
For example, nvim-cmp shouldn't filter results 7 times if the user press `console` very fast.
If the user specify `throttle_time` as 120(ms), nvim-cmp will filter results each 70ms.